docs/docker-integration/install-node.rst

```diff
@@ -21,6 +21,12 @@ Installing the Flocker Node Services
 * :ref:`aws-install-docker`
 * :ref:`rackspace-install-docker`
 
+.. _rhel-7-install-docker:
```

Comment:
Does this need to be replicated in the Kubernetes and Standalone instructions? Currently the RHEL instructions for Installing the Node services are only appearing in the Docker integration instructions.